tests/topotests/bgp_vpnv4_noretain/test_bgp_vpnv4_noretain.py

```diff
@@ -135,12 +135,12 @@ def teardown_module(_mod):
 
 def router_json_cmp_exact_filter(router, cmd, expected):
     # filter out tableVersion, version and nhVrfID
-    output = router.cmd('vtysh -c "{}" | grep -v ersion | grep -v nhVrfId'.format(cmd))
```

Comment:
I asked 2 weeks ago regarding this: https://github.com/FRRouting/frr/pull/12685#pullrequestreview-1267119126

Seems @louis-6wind wanted this to be _EXACTLY_ matched, but not sure what was the reason for not being partial. @louis-6wind could you elaborate?

Reply:
Please this fix instead https://github.com/FRRouting/frr/pull/12768

The test is checking that some prefixes are actually filtered from the BGP tables. To do so it needs to match the exact output. Otherwise, the test says OK even if the prefix that was expected to be filtered is in the output.